kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -160,8 +244,8 @@ define([
                 // This row is already open - close it
                 dtRow.child.hide();
                 $currentRow.removeClass('vertical_collapse--open');
-                if (logWidgets[jobId]) {
-                    logWidgets[jobId].stop();
+                if (widgetsById[jobState.job_id]) {
```

Comment:
changed `logWidgets` => `widgetsById` since there will only be log widgets now, no job state list row widgets